doc/docs-revamp/dev-onboarding-quick-setup.md

```diff
@@ -0,0 +1,375 @@
+---
+title: "Developer onboarding and quick setup guide"
+description: "setting up and testing developer environment, writing first smart contract"
+date: 2024-03-26
+---
+
+# Section 3. Developer onboarding and quick setup guide
+
+This guide's objective is to help you set up your development environment, test it, and write your first smart contract within an hour. This will be a simple vesting contract in which someone sends a gift of ada to a beneficiary. The beneficiary can access their gift only after a specific deadline has passed and the beneficiary has signed the contract. 
+
+## Setting up and testing your development environment
+
+### Prerequisites
+
+#### Hardware and OS requirements
+
+**Processing power**
+- Minimum: Intel Core i5 or AMD equivalent
+- Recommended: Intel Core i7 or AMD Ryzen 7 
+
+**RAM**
+- Minimum: 8GB RAM
+- Recommended: 16GB RAM or more
+
+**Operating system**
+- Linux: A recent distribution of Linux such as Ubuntu 20.04 LTS or later is preferred due to its widespread use in the Haskell community and its compatibility with Plutus tooling.
+- macOS: macOS Catalina (10.15) or later
+- Windows: While Windows can be used for Haskell development, it is less common and might require additional setup such as using Windows Subsystem for Linux (WSL2) to provide a Linux-compatible environment.
+
+### Tools
```

Comment:
I suggest stripping out this whole subsection. This alone is enough: https://github.com/IntersectMBO/plutus/blob/master/doc/read-the-docs-site/quick-start.rst

Reply:
Looks like some great resources in that quick-start.rst file!